### PR TITLE
Update configuration.md

### PR DESCRIPTION
**Pull request by carriecrowe1138**
Corrected several typos and included words that were originally left out to ensure proper grammar in document.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/administration/configuration.md [hunk 1]
```diff
@@ -292,7 +291,9 @@ The database user's password (not applicable for `sqlite3`). If the password con
 ### url
 
 Use either URL or the other fields below to configure the database
-Example: `mysql://user:secret@host:port/database`
+Example: `mysql://user:secret@
```
Comment (osg-grafana) [inline docs/sources/administration/configuration.md:294]: Needs engineering input because this looks like code style.

**Comment by osg-grafana** [comment]
Verbiage as of this comment as approved from the doc side. Thank you, @carriecrowe1138!

File: docs/sources/administration/configuration.md (hunk 1 above)
Comment (hairyhenderson) [reply docs/sources/administration/configuration.md:294]: This is intended to be a URL, so must remain on one line.

File: docs/sources/administration/configuration.md [hunk 2]
```diff
@@ -292,7 +291,9 @@ The database user's password (not applicable for `sqlite3`). If the password con
...
+
+:port/database`
```
Comment (hairyhenderson) [inline docs/sources/administration/configuration.md:296]: Not sure why `host` disappeared, but it's an important part of the URL 🙂

```suggestion
Example: `mysql://user:secret@host:port/database`
```

File: docs/sources/administration/configuration.md [hunk 3]
```diff
@@ -202,15 +202,14 @@ Or redirect port 80 to the Grafana port using:
 $ sudo iptables -t nat -A PREROUTING -p tcp --dport 80 -j REDIRECT --to-port 3000
 ```
 
-Another way is put a webserver like Nginx or Apache in front of Grafana and have them proxy requests to Grafana.
+Another way is to put a web server like Nginx or Apache in front of Grafana and have them proxy requests to Grafana.
 
 ### domain
 
-This setting is only used in as a part of the `root_url` setting (see below). Important if you use GitHub or Google OAuth.
```
Comment (hairyhenderson) [inline docs/sources/administration/configuration.md:209]: I'm not clear why this line was deleted - was this intentional?

Comment (osg-grafana) [reply docs/sources/administration/configuration.md:209]: Thanks for catching that deletion. I am also unclear on this.

**Review by osg-grafana — DISMISSED** [review]
Requesting changes based on technical input.

File: docs/sources/administration/configuration.md (hunk 3 above)
Comment (achatterjee-grafana) [reply docs/sources/administration/configuration.md:209]: Do we have definitive guidance on using "webserver" versus "web server" in our style guide? If not, we should probably add instructions to use "web server" and not "webserver".

File: docs/sources/administration/configuration.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/administration/configuration.md:294]: +1 to @hairyhenderson's comment.

File: docs/sources/administration/configuration.md (hunk 3 above)
Comment (hairyhenderson) [reply docs/sources/administration/configuration.md:209]: @achatterjee-grafana was that comment intended for [line 205](https://github.com/grafana/grafana/pull/30695/files#diff-d0b4774dfc86c7e56111aa3e75307f80bccde523195641b239ce17051880ff2aR205) instead?

Comment (achatterjee-grafana) [reply docs/sources/administration/configuration.md:209]: Lime 205 - "web server".

Comment (osg-grafana) [reply docs/sources/administration/configuration.md:209]: Currently, there is no entry for that in the style guide, and I too vote to add it as web server. Rationale being that web page is not written webpage, and web browser is not webbrowser. Will handle that in a subsequent PR.

Comment (hairyhenderson) [reply docs/sources/administration/configuration.md:209]: @osg-grafana @achatterjee-grafana - can you comment on ln205 about that instead? this comment thread is for ln209 😉

File: docs/sources/administration/configuration.md [hunk 4]
```diff
@@ -202,15 +202,14 @@ Or redirect port 80 to the Grafana port using:
 $ sudo iptables -t nat -A PREROUTING -p tcp --dport 80 -j REDIRECT --to-port 3000
 ```
 
-Another way is put a webserver like Nginx or Apache in front of Grafana and have them proxy requests to Grafana.
+Another way is to put a web server like Nginx or Apache in front of Grafana and have them proxy requests to Grafana.
```
Comment (osg-grafana) [inline docs/sources/administration/configuration.md:205]: Currently, there is no entry for that in the style guide, and I too vote to add it as web server. Rationale being that web page is not written webpage, and web browser is not webbrowser. Will handle that in a subsequent PR.

Comment (achatterjee-grafana) [reply docs/sources/administration/configuration.md:205]: Moving my comment to in here. Do we have definitive guidance on using "webserver" versus "web server" in our style guide? If not, we should probably add instructions to use "web server" and not "webserver".

I will update the style guide.

**Comment by osg-grafana** [comment]
> [CLA assistant check](https://cla-assistant.io/grafana/grafana?pullRequest=30695)
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=30695) before we can accept your contribution.
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/grafana?pullRequest=30695) it.

@carriecrowe1138, can you sign the CLA?

**Review by oddlittlebird — APPROVED** [review]
LGTM. Thanks for contributing to the docs!

**Pull request merged**